src/components/questionary/questionaryComponents/SampleDeclaration/SampleDeclaratonDefinition.tsx

```diff
@@ -27,7 +26,7 @@ export const sampleDeclarationDefinition: QuestionaryComponentDefinition = {
   },
   createYupValidationSchema: (answer) => {
     const config = answer.config as SubtemplateConfig;
-    let schema = Yup.array().of(Yup.object<SampleWithQuestionary>());
```

Comment:
Do you know if Yup removed a possibility to type Array?

Reply:
I think it's not removed but I guess they have improved some types and it makes some problems with our generated types where we have those properties like __typename and so on and the easiest solution was to just use AnyObjectSchema for now. Maybe it can be resolved more elegant later but it needs some more investigation how to do it.